pkg/utils/apiresources.go

```diff
@@ -162,9 +175,9 @@ func (r *DisabledResourceConfig) parseSingle(token string) error {
 	return nil
 }
 
-// IsResourceDisabled returns whether a given GroupVersionKind is disabled.
-// a gkv is disabled if its group or group version is disabled
-func (r *DisabledResourceConfig) IsResourceDisabled(gvk schema.GroupVersionKind) bool {
```

Comment:
Should we keep the original interface?

you can have something like, and then you don't need to update the caller code.

```
func (r *ResourceConfig) IsResourceDisabled(gvk schema.GroupVersionKind) bool {
	isConfigured := r.isResourceConfigured(gvk)
	if r.isAllowedList {
		return !isConfigured
	}
	return isConfigured
}
```
and change the "IsResourceConfigured" to "isResourceConfigured", which is more like an internal util func.